packages/react-storage/src/storage-browser.css

```diff
@@ -0,0 +1,51 @@
+.amplify-storagebrowser__header {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
```

Comment:
Are these hardcoded `gap` values just place holders?

Reply:
Mostly a placeholder; I could make them a css property (`var(--gap)`) and use a REM value like we do in UI.

Reply:
Updated in latest to use css properties